targets/targets.json

```diff
@@ -3029,6 +3029,31 @@
         "device_name": "STM32H743ZI",
         "bootloader_supported": true
     },
+    "UHURU_RAVEN": {
+        "inherits": ["FAMILY_STM32"],
```

Comment:
I advice to add
"device_has_remove": ["SERIAL_FC"],
as you don't set any pins in PinMap_UART_RTS and PinMap_UART_CTS